src/main/java/org/opensearch/performanceanalyzer/rca/store/rca/cache/FieldDataCacheRca.java

```diff
@@ -214,6 +219,18 @@ public void setCollectorTimePeriod(long metricTimePeriodInMillis) {
         public void collect(final long currTimestamp) {
             for (MetricFlowUnit flowUnit : cacheEvictionMetrics.getFlowUnits()) {
                 if (flowUnit.isEmpty() || flowUnit.getData() == null) {
+                    clearCounter += 1;
+                    if (clearCounter > consecutivePeriodsToClear) {
+                        // If the RCA receives 3 empty flow units, re-set the 'hasMetric' value
+                        hasEvictions = false;
+                        clearCounter = 0;
+                        LOG.error(
```

Comment:
For some reason if there are only empty flowunits, this might flood the logs. As we are emitting a metric, can we modify the log level to debug?

Reply:
Good point, let me change the log level to debug.